src/main/scala/org/globalforestwatch/features/FireAlertsViirsFeature.scala

```diff
@@ -8,10 +8,32 @@ import org.globalforestwatch.util.GeometryReducer
 object FireAlertsViirsFeature extends Feature {
   override val geomPos: Int = 0
 
+  override def isValidGeom(i: Row): Boolean = {
+    val lon = i.getString(geomPos + 1).toDouble
+    val lat = i.getString(geomPos).toDouble
+
+    lat >= -90 && lat <= 90 && lon >= -180 && lon <= 180
+  }
+
   override def get(i: Row): vector.Feature[Geometry, FeatureId] = {
     val featureId = getFeatureId(i)
+    val lon = i.getString(geomPos + 1).toDouble
+    val lat = i.getString(geomPos).toDouble
+
+    // if lat or lon is whole number, it might be on border of geotrellis grid cell,
+    // so push it slightly to south or east cell (since cells are inclusive on northwest)
+    val adjustedLon =
+      if (lon.toString.split("[.]")(1).size == 1)
```

Comment:
Codacy found an issue: [The string literal '[.]' appears 2 times in the file.](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5902460)